cli.js

```diff
@@ -12,7 +14,7 @@ const {
 const cli = meow(`
   Usage
 
-    $ bpmn-to-image <diagramFile>:<outputConfig> ...
+    $ bpmn-to-image <diagramFile>${path.delimiter}<outputConfig> ...
```

Comment:
One additional thing: Is there any chance to store the value of `path.delimeter` in a first place and then re-use it, instead of always getting it directly. Just in terms of code quality

Reply:
I rework it to do in same way as tests. Let me know if that works. Done